cob_twist_controller/scripts/test/test_careobot_st_jla_ca_sphere.py

```diff
@@ -157,23 +159,21 @@ def init_pos():
             # save data
             for data_kraken in data_krakens:
                 data_kraken.writeAllData()
-        except rospy.ROSInterruptException as e:
-            rospy.logwarn('ROSInterruptException: ' + str(e))
         except:
             rospy.logerr('Else exception.')
         else:
             for data_kraken in data_krakens:
                 data_kraken.writeAllData()
 
         try:
-            # pid.send_signal(subprocess.signal.SIGINT)
+            # pid.send_signal(signal.SIGINT)
             pid.kill()
-            pid.send_signal(subprocess.signal.SIGINT)
+            pid.send_signal(signal.SIGINT)
         except Exception as e:
             rospy.logerr('Failed to stop rosbag play due to exception: ' + str(e))
         try:
             traj_marker_pid.kill()
-            traj_marker_pid.send_signal(subprocess.signal.SIGINT)
+            traj_marker_pid.send_signal(subprocess32.signal.SIGINT)
```

Comment:
`subprocess32` ? Where does that come from?

Reply:
was a test...I reverted it already again - but only in #227...now both PRs are in sync again